### PR TITLE
Renewal-line icon

### PR DESCRIPTION
**Pull request by habin-isa**
## What does this do?
- Adds new icon `renewal-line`
- Updates `CHANGELOG`

### CODE REVIEW

**Review by JamieMarsay — APPROVED** [review]

File: src/Icon/Icon.tsx [hunk 1]
```diff
@@ -1366,6 +1366,25 @@ export const Icon: FC<Props> = ({
...
+        >
+          <path
+            fillRule="evenodd"
+            clipRule="evenodd"
+            d="M20.75 2C20.75 1.447 21.198 1 21.75 1C22.302 1 22.75 1.447 22.75 2V9C22.75 9.553 22.302 10 21.75 10H14.75C14.198 10 13.75 9.553 13.75 9C13.75 8.447 14.198 8 14.75 8H19.458C17.81 4.921 14.768 3 11.375 3C7.51199 3 4.10199 5.49 2.68899 9.344C2.49899 9.862 1.92599 10.13 1.40599 9.938C0.886985 9.749 0.620985 9.174 0.810985 8.655C2.51599 4.005 6.66299 1 11.375 1C15.213 1 18.676 2.995 20.75 6.248V2ZM20.811 14.656C21.001 14.138 21.575 13.871 22.094 14.062C22.613 14.251 22.879 14.826 22.689 15.345C20.984 19.995 16.837 23 12.125 23C8.28698 23 4.82398 21.005 2.74998 17.752V22C2.74998 22.553 2.30198 23 1.74998 23C1.19798 23 0.749985 22.553 0.749985 22V15C0.749985 14.447 1.19798 14 1.74998 14H8.74998C9.30198 14 9.74998 14.447 9.74998 15C9.74998 15.553 9.30198 16 8.74998 16H4.04198C5.68998 19.079 8.73199 21 12.125 21C15.988 21 19.398 18.51 20.811 14.656Z"
+            fill={theme.colors[color]}
+          />
+        </svg>
+      )
+    }
```
Comment (fleonard) [inline src/Icon/Icon.tsx:1387]: don't want to be annoying but any chance you can keep this file in alphabetical order? 🙏🏻

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## [0.67.2] - 2021-09-14
+### Added
+- Added new Icon svg for Renewal-Line
+
```
Comment (fleonard) [inline CHANGELOG.md:4]: this looks wrong, new version should be 1.0.something - you might need to pull master and sort this out

**Comment by fleonard** [comment]
@habin-isa if rebasing it's gonna be a mess just close this PR and open a new one since it's just one new icon

**Comment by habin-isa** [comment]
Rebasing was tangled so opening in new PR!

**Pull request closed without merging**